projects/core/src/auth/store/effects/open-id-token.effect.ts

```diff
@@ -1,12 +1,42 @@
 import { Injectable } from '@angular/core';
 import { Actions, Effect, ofType } from '@ngrx/effects';
 import { Observable, of } from 'rxjs';
-import { catchError, exhaustMap, map } from 'rxjs/operators';
+import {
+  catchError,
+  exhaustMap,
+  filter,
+  map,
+  withLatestFrom,
+} from 'rxjs/operators';
+import { AuthConfig } from '../../config/auth-config';
 import { OpenIdAuthenticationTokenService } from '../../services/open-id-token/open-id-token.service';
 import * as fromActions from '../actions/open-id-token.action';
+import {
+  LoadUserToken,
+  LOAD_USER_TOKEN,
+  LOAD_USER_TOKEN_SUCCESS,
+} from '../actions/user-token.action';
 
 @Injectable()
 export class OpenIdTokenEffect {
+  @Effect()
+  triggerOpenIdTokenLoading$: Observable<
+    fromActions.LoadOpenIdToken
+  > = this.actions$.pipe(
+    filter(
+      _ => this.config.authentication && this.config.authentication.kyma_enabled
```

Comment:
`iff` operator would be much more efficient here

Reply:
Done, thanks